mp_cnn/README.md

```diff
@@ -39,7 +43,23 @@ python main.py mpcnn.msrvid.model.castor --dataset msrvid --batch-size 16 --epsi
 | Implementation and config        | Pearson's r   |
 | -------------------------------- |:-------------:|
 | Paper                            | 0.9090        |
-| PyTorch using above config       | 0.9050        |
+| PyTorch using above config       | 0.8911        |
+
+## TrecQA Dataset
+
+To run MP-CNN on (Raw) TrecQA, you first need to the `get_trec_eval.sh` script in `utils` under the repo root.
```

Comment:
I'd suggest writing the actual command so that a user can directly copy paste it into his terminal. Also add commands to uncompress and make `trec_eval`.

Reply:
Yep, my sentence is also incoherent :)